roles/wireguard/templates/server.conf.j2

```diff
@@ -0,0 +1,18 @@
+[Interface]
+Address = {{ wireguard_network_ipv4['subnet'] }}/{{ wireguard_network_ipv4['prefix'] }}{% if ipv6_support %},{{ wireguard_network_ipv6['gateway'] }}/{{ wireguard_network_ipv6['prefix'] }}
+{% endif %}
+
+DNS = {{ local_service_ip }}
+ListenPort = {{ wireguard_port }}
+PrivateKey = {{ lookup('file', wireguard_config_path + '/private/' + IP_subject_alt_name) }}
+SaveConfig = false
```

Comment:
You may want to change this to true, so that when restarting, if a packet hits the server to be forwarded to a client, the server will re-initiate the session with the client using the client's recent ephemeral port if it's still open.

Reply:
Any saved config will be rewritten is someone want to update users on the servers. Configuration management is done via Ansible

Reply:
That's fine. The purpose of adding saveconfig=true is so that when an update brings down the interface and brings it back up, the ephemeral endpoints (and therefore the connections) are restored, and the user's downloads continue as if nothing happened.